Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM bitnami/node:8
-#FROM circleci/node:8-stretch-browsers
+# FROM bitnami/node:8
+FROM circleci/node:8
```

Comment:
This is the Dockerfile used to actually build and run the production app, right? Isn't it strange to base it on Circle's, which presumably includes a lot of stuff that's unnecessary in prod?

Reply:
Yep, it was a failed experiment. I didn't understand what the hokusai setup was doing. @ashfurrow helped me understand it a bit better. Ultimately I think I'll need to learn a little more before I can take this anywhere. 

Right now it _seems_ impossible to use circleci's caching for the test step (which is by far the longest). It's also going through the process of downloading yarn every time. I'll be reverting this change, but I'm curious if we can take the test step off of the hokusai image.... if not that, I think optimizing the force build is going to be a really, really hard process.